iothub/device/src/PropertyConvention.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Since we haven't split the conventions up we should get rid of this correct?

We should move the component* items to a constants class.